### PR TITLE
moving chainconfig to env

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -9,3 +9,21 @@ REACT_APP_SUPPORT_URL=
...
 SENTRY_AUTH_TOKEN=
+
+## Chainbridge configuration
+### Ethereum 
+REACT_APP_ETHEREUM_CHAIN_ID="5"
+REACT_APP_ETHEREUM_NETWORK_ID="5"
+REACT_APP_ETHEREUM_NETWORK_NAME="Ethereum Goerli"
+REACT_APP_ETHEREUM_BRIDGE_ADDRESS="0xa806cA3bD88F790744462cBC34c40EDd5b8dc2Dd"
+REACT_APP_ETHEREUM_ERC20_HANDLER_ADDRESS="0xf934Bfc8B5241b6C9e0DfC9A329AD687e79c5498"
+REACT_APP_ETHEREUM_RPC_URL="https://goerli.infura.io/v3/2532f2073c2d4a34b5b9544c76a3de8b"
```
Comment (AndreiNavoichyk) [inline .env.example]: We don't need this, remove

File: .env.example [hunk 2]
```diff
@@ -9,3 +9,21 @@ REACT_APP_SUPPORT_URL=
 REACT_APP_SENTRY_DSN_URL=
 REACT_APP_SENTRY_ENV=development
 SENTRY_AUTH_TOKEN=
+
+## Chainbridge configuration
+### Ethereum 
+REACT_APP_ETHEREUM_CHAIN_ID="5"
```
Comment (AndreiNavoichyk) [inline .env.example:15]: What is CHAIN ID?

File: .env.example [hunk 3]
```diff
@@ -9,3 +9,21 @@ REACT_APP_SUPPORT_URL=
 REACT_APP_SENTRY_DSN_URL=
 REACT_APP_SENTRY_ENV=development
 SENTRY_AUTH_TOKEN=
+
+## Chainbridge configuration
+### Ethereum 
+REACT_APP_ETHEREUM_CHAIN_ID="5"
+REACT_APP_ETHEREUM_NETWORK_ID="5"
```
Comment (AndreiNavoichyk) [inline .env.example:16]: What is NETWORK ID?

File: .env.example [hunk 4]
```diff
@@ -9,3 +9,21 @@ REACT_APP_SUPPORT_URL=
...
+REACT_APP_ETHEREUM_NETWORK_NAME="Ethereum Goerli"
+REACT_APP_ETHEREUM_BRIDGE_ADDRESS="0xa806cA3bD88F790744462cBC34c40EDd5b8dc2Dd"
+REACT_APP_ETHEREUM_ERC20_HANDLER_ADDRESS="0xf934Bfc8B5241b6C9e0DfC9A329AD687e79c5498"
+REACT_APP_ETHEREUM_RPC_URL="https://goerli.infura.io/v3/2532f2073c2d4a34b5b9544c76a3de8b"
+REACT_APP_ETHEREUM_ERC20_TOKEN_ADDRESS="0xb8A59CEF67d12C5C75836aEfF1d97943F5A9F662"
+REACT_APP_ETHEREUM_ERC20_NAME="CN"
+REACT_APP_ETHEREUM_ERC20_SYMBOL="CS"
+
+### Cerebellum Network
+REACT_APP_CERE_CHAIN_ID="1"
```
Comment (AndreiNavoichyk) [inline .env.example]: What is CHAIN ID?

File: .env.example [hunk 5]
```diff
@@ -9,3 +9,21 @@ REACT_APP_SUPPORT_URL=
...
+REACT_APP_CERE_NETWORK_ID="2"
```
Comment (AndreiNavoichyk) [inline .env.example]: What is NETWORK ID?

File: .env.example (hunk 2 above)
Comment (karthikeyan-cere) [reply .env.example:15]: It is the Ethereum Chain Id. For each chain, it will vary.

File: .env.example (hunk 3 above)
Comment (karthikeyan-cere) [reply .env.example:16]: The network id is a unique Id for all networks with Ethereum. For Goerli it is 5

File: .env.example (hunk 4 above)
Comment (karthikeyan-cere) [reply .env.example]: Cerebellum chain ID

File: .env.example (hunk 5 above)
Comment (karthikeyan-cere) [reply .env.example]: Cerebellum Network Id

File: .env.example (hunk 1 above)
Comment (karthikeyan-cere) [reply .env.example]: done

File: .env.example (hunk 4 above)
Comment (AndreiNavoichyk) [reply .env.example]: Where to take it?

File: .env.example (hunk 5 above)
Comment (AndreiNavoichyk) [reply .env.example]: Where to take it?

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**